CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
+1.29
```

Comment:
Not 1.28?

Reply:
1.28 is out already - I forgot to update the CHANGELOG because there weren't any really significant changes. But I should indeed fix that. But this is correct